Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);

```

Comment:
**Suggestion**: Reuse the Scanner instance from the main method instead of creating a new instance in the Even method to reduce redundancy and improve resource management. <br> **Code Suggestion**: 
 ```
 +    // Consider passing the Scanner instance from the main method to avoid redundancy
 -        Scanner sc = new Scanner(System.in);
 ```